ui/models.py

```diff
@@ -230,6 +230,20 @@ def transcoded_videos(self):
         return sorted(self.videofile_set.exclude(encoding=EncodingNames.ORIGINAL),
                       key=lambda x: EncodingNames.MP4.index(x.encoding) if x.encoding in EncodingNames.MP4 else 0)
 
+    @property
+    def download(self):
+        """
+        Return the most appropriate videofile for downloading.
+
+        Returns:
+            VideoFile: The videofile most appropriate for download (highest quality MP4 transcode or original upload)
+        """
+        files = sorted(self.videofile_set.exclude(encoding=EncodingNames.HLS),
```

Comment:
If this is meant to only include MP4 related encodings should we use `encoding__in=EncodingNames.MP4` here instead?

Reply:
It needs to include `EncodingNames.ORIGINAL` too (not in `MP4`).